scripts/exglobal_forecast.sh

```diff
@@ -434,13 +445,16 @@ if [ $cplwav = ".true." ]; then
     if [ $RERUN = "NO" ]; then
       if [ ! -f ${WRDIR}/${sPDY}.${scyc}0000.restart.${wavGRD} ]; then 
         echo "WARNING: NON-FATAL ERROR wave IC is missing, will start from rest"
+      else
+        $NLN ${WRDIR}/${sPDY}.${scyc}0000.restart.${wavGRD} $DATA/restart.${wavGRD}
       fi
-      $NLN ${WRDIR}/${sPDY}.${scyc}0000.restart.${wavGRD} $DATA/restart.${wavGRD}
     else
       if [ ! -f ${RSTDIR_WAVE}/${PDYT}.${cyct}0000.restart.${wavGRD} ]; then
-        echo "WARNING: NON-FATAL ERROR wave IC is missing, will start from rest"
+        echo "ERROR: wave IC is missing for RERUN, exiting"
```

Comment:
@aerorahul and I discussed this.  We believe that the model should have failed in a situation that we had last week.  In a RERUN, we should have a wave IC and not start from rest.  We should never get to this "failure" now because of the first chunk of code, but think it's best to have this failure alerted in an odd chance we do.

Reply:
If it ever fails, what should we do about it? Should we apply any special procedures to recover? If so, can the script be smarter to do it for us?

If it's a rerun, there is a failure. I think it would be better to just start from rest to minimize the delay and a warning message would be enough.

Reply:
We should restart from FHR 0 if it fails here.  Starting from rest gets you results faster, but the quality of those results would not be good for the wave model.

Reply:
Can we restart from FHR 0 if it fails instead of exiting?

Reply:
The logic here: https://github.com/JessicaMeixner-NOAA/global-workflow/blob/bug/waverstops/scripts/exglobal_forecast.sh#L171-L204 means that if there isn't a valid WW3 rerun restart, RERUN=NO and we would start at 0, meaning you should never get to this chunk of code.

Reply:
Sorry I am having a hard time understanding why there would be a fatal that we don't expect at all and we would have no idea how to recover from it.

Reply:
I guess my question to that would be, why would you not want to fail if for some reason you got to this point?  In terms of recovering, the solution would be to run again, but again I don't know how you get to this point after the above logic.   If @aerorahul agrees, we can revert back to a warning, but I'd prefer to keep the change for a fatal error.

Reply:
Okay let me ask this....What would happen if we had such changes in production last week? Would it have proceeded with RERUN=NO or failed with RERUN=YES? Proceeded with NO I assume, and that's all we need.

Now to the details...Why do we want to check the same files twice? [It](https://github.com/JessicaMeixner-NOAA/global-workflow/blob/bug/waverstops/scripts/exglobal_forecast.sh#L452-L454) was used to check availability, but with the newly added logic, we won't get to this point. If we won't, why it's still in the script? I am fine with a fatal, but I still feel it's unnecessary in this case. 

Maybe we can set up a 15-30 minutes meeting next week to discuss this.

Thanks!

Reply:
If this happened in production last week, the first chunk of code in this PR https://github.com/JessicaMeixner-NOAA/global-workflow/blob/bug/waverstops/scripts/exglobal_forecast.sh#L171-L204
would have ensured that we went back to a start time for the restart that we also had a wave restart for.  So RERUN=YES, but it would have been an earlier restart time.  This error statement would never be invoked with the first section of code. 

I'm happy to meet next week to discuss further.

Reply:
We could re-work this logic as follows:
```
if [ -f ${RSTDIR_WAVE}/${PDYT}.${cyct}0000.restart.${wavGRD} ]; then
   $NLN ${RSTDIR_WAVE}/${PDYT}.${cyct}0000.restart.${wavGRD} $DATA/restart.${wavGRD}
fi
```
There is no condition for this file to not be present given the checks above.

Reply:
That works for me.  @WeiWei-NCO if that is also okay with you I can update this branch accordingly.

Reply:
Works for me too. Thanks, Jessica and Rahul!